app/build.gradle

```diff
@@ -0,0 +1,42 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion versions.androidCompileSdk
+    buildToolsVersion versions.androidBuildTools
+    defaultConfig {
+        applicationId "com.ustwo.boilerplate"
+        minSdkVersion versions.androidMinSdk
+        targetSdkVersion versions.androidTargetSdk
+        versionCode 1
+        versionName "1.0"
+        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
```

Comment:
should we do a `com.ustwo.boilerplate.BoilerPlateEspressoTestRunner` just in case we want to test our Applications ? this is up for discussion.